### PR TITLE
Complete and convert India parser to CSV

### PR DESCRIPTION
**Pull request by axmb**

### CODE REVIEW

File: ingestion/functions/parsing/india/india.py [hunk 1]
```diff
@@ -32,70 +33,116 @@ def convert_gender(raw_gender):
...
     if city:
-        location["administrativeAreaLevel3"] = city
         query_terms = (city,) + query_terms
+        resolutions = ("Admin3",) + resolutions
 
-    location["query"] = ", ".join(query_terms)
-    return location
+    return {
+        "query": ", ".join(query_terms),
+        "limitToResolution": ",".join(resolutions)
```
Comment (attwad) [inline ingestion/functions/parsing/india/india.py:57]: you already add `,` to the resolutions above, it'd simpler if you just added the resolution names in an array and joined them here as you do

File: ingestion/functions/parsing/india/india.py [hunk 2]
```diff
@@ -32,70 +33,116 @@ def convert_gender(raw_gender):
...
-                },
-                "notes": entry["notes"] or None
-            } for entry in cases["raw_data"] if entry["agebracket"])
+                "demographics": convert_demographics(row),
+                "notes": row["Notes"] or None
+            }
+            update_for_status(row, case)
+            for i in range(int(row["Num Cases"])):
+                c = copy.deepcopy(case)
+                c["caseReference"]["sourceEntryId"] = f"{uuid_prefix}{row[uuid_column]}-{i + 1}"
```
Comment (attwad) [inline ingestion/functions/parsing/india/india.py:144]: "uuid"... you had one job!    ;-)

**Review by attwad — APPROVED** [review]

File: ingestion/functions/parsing/india/india.py (hunk 1 above)
Comment (axmb) [reply ingestion/functions/parsing/india/india.py:57]: SG, done.

File: ingestion/functions/parsing/india/india.py (hunk 2 above)
Comment (axmb) [reply ingestion/functions/parsing/india/india.py:144]: (╯°□°)╯︵ ┻━┻

**Pull request merged**